### PR TITLE
UI for CookRecipe

### PR DESCRIPTION
**Pull request by azhara-a**
- Remove download button from audio
- Move modal prompt("Do you want the recipe to be read out loud?") to tutorial tab
- Move speaker button next to title
Mobile:
image
Desktop:
Screenshot 2020-09-15 at 11 34 03 AM
image

### CODE REVIEW

**Review by mam249 — COMMENTED** [review]
LGTM

**Comment by lminchillo** [comment]
>   Screenshot 2020-09-14 at 7 43 28 PM

The button design confuses me a bit. When it's saying "Always read steps" does it mean that it _is_ reading the steps or does it mean that I should click it in order to have it read the steps? Perhaps a checkbox might be more intuitive?

That being said, I'd also prefer it if the items were a bit more spaced in the button, something like this:
image

File: frontend/src/components/individual-recipe/CookRecipe.js [hunk 1]
```diff
@@ -205,5 +226,28 @@ class CookRecipe extends Component {
...
+      this.pauseAudio();
+    }
+  }
+
+  getSpeakerIcon() {
+    return this.state.isSpeakerOn ? speakerOff : speakerOn;
+  }
+
+  getSpeakerMessage() {
+    return this.state.isSpeakerOn ? "Don't read steps" : "Always read steps";
```
Comment (lminchillo) [inline frontend/src/components/individual-recipe/CookRecipe.js:250]: I think `Always` is not needed - you could just say `Read steps`

**Comment by azhara-a** [comment]
> That being said, I'd also prefer it if the items were a bit more spaced in the button, something like this:
Updated desktop and mobile screenshots.

> The button design confuses me a bit. When it's saying "Always read steps" does it mean that it _is_ reading the steps or does it mean that I should click it in order to have it read the steps? Perhaps a checkbox might be more intuitive?
When it says "read steps" it means that it's not reading them, but if the user wants the steps to be read out loud they need to press on it and then the audio will come up and the button will change to "don't read steps".

**Review by mugdhalakhani — APPROVED** [review]
LGTM, thanks for implementing my suggestions! :)

**Review by lminchillo — APPROVED** [review]

**Pull request merged**